ray-operator/config/default/kustomization.yaml

```diff
@@ -9,10 +9,10 @@ namespace: ray-system
 #namePrefix: ray-operator-
 
 # Labels to add to all resources and selectors.
-#commonLabels:
+#labels:
 #  someName: someValue
 
```

Comment:
ditto

Reply:
@MortalHappiness thanks for the review, there were no specific reasons not to use the `kustomize edit fix` command directly so after the review I used it to fix those deprecated values/ fix missing fields